content/blog/posts/A-Message-from-the-Director.rst

```diff
@@ -0,0 +1,66 @@
+A Message from the Director
+===========================
+:date: 2017-08-15
+:slug: a-message-from-the-director
+:author: Lance Albertson
+:img: OSLSignPicture.jpg
+:tag: featured-stories
+
+Since its creation in 2002, the Open Source Lab (OSL) has been a beacon of
+innovation, experiential learning, and hosting at Oregon State University and
+beyond. As director, I’ve had the pleasure of watching the lab and its employees
+grow over the years and I’m so grateful to have been given such an opportunity.
+I want to express why the OSL is important not only to the open source
+community, but to our larger academic institution and its students.
+
+Becoming director of the OSL has been one of the most rewarding positions I have
+ever had the pleasure of filling. I have had the opportunity to watch students
+with amazing potential grow into graduates with incredible opportunities, all
+because of their hard work. As much as I would like to say the OSL is
+responsible for this transformation, I must give credit where credit is due:
+The students who come through the OSL are passionate, driven, and highly
+intelligent. They’re intelligent, patient, and generous in time and knowledge.
```

Comment:
"They’re intelligent, patient, and generous in time and knowledge."

to

"They’re dedicated, patient, and generous in sharing their time and knowledge."